### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by veronica365**
### Description

- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for your reusable components: /components.
- Install [React Router](https://reactrouter.com/en/main/start/tutorial#setup)
[Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App>
- Add 2 routes in your application:
  - `/`, which contains 
    - A book list, where each book has a title and author
    - A form that allows for creating a new book
   
  - `/categories`
    -  This should display a button that checks for a status
- Include a navigation component that includes links to the 2 routes you created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,170 @@
...
+
+- [ ] Add more tests and improve the application CSS styles
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/veronica365/bookstore-react/issues).
```
Comment (Ismailco) [inline README.md]: - [ ] Kindly make sure to fix the `issues` link, it is broken and redirects to a 404 page

File: src/components/Book/BookItem/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default function Book({ book }) {
+  return (
+    <div>
+      <h3>{book.title}</h3>
+      <p>{book.author}</p>
+    </div>
```
Comment (Ismailco) [inline src/components/Book/BookItem/index.js]: - [ ] This component should have a button to delete a book from the Books list, kindly make sure to implement this so the books can be deleted

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @veronica365, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Clean code
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — APPROVED** [review]
Hi @veronica365 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by veronica365** [comment]
Thanks for the review

**Pull request merged**